docs/organizations/managing-security-and-risk.md

```diff
@@ -1,73 +1,131 @@
 # Managing security and risk
 
-The Security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your [connected Jira instance](./integrations/jira-integration.md).
+The Security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security across your organization by automatically opening time-bound, prioritized findings whenever security problems are detected in your organization repositories, in your [connected Jira instance](./integrations/jira-integration.md), or during [penetration testing](https://go.codacy.com/pen-testing-product)
 
-Under Security and risk management, you can find the following pages to help you monitor your security issues:
+Under Security and risk management, you can find the following pages to help you monitor security:
 
--   [Dashboard](#dashboard)
--   [Item list](#item-list)
+-   [Overview](#dashboard)
+-   [Findings](#item-list)
 
```

Comment:
IMO, this approach of having a "table of contents" in the introduction for the main areas only (and not including all the sections) it's much cleaner and reads better. 👍 
Though I was expecting here that sentence you intended to add, mentioning the possible actions (sharing and exporting). I believe it would fit great.